CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ set(PROJECT_NAME
 
 project(${PROJECT_NAME})
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
standard 14 includes standard 11 and have newer options so it is even better to have 14 then